src/services/Queries.js

```diff
@@ -408,5 +408,78 @@ class Queries {
       return error;
     }
   }
+
+  /** Function to find a user with a manager role
+   *
+   * This service get a spacific trip that corresponds to user id and accomodation id
+   * @param {object} table table to be searching from
+   * @param {Object} userid user id
+   * @param {Object} accommodationid accomodation id
+   * @returns {object} data of the trip found that corresponds to that user and accomodation id
+   */
+  static async findIfAccomodationBooked(table, userid, accommodationid) {
```

Comment:
@mujohn26 What this function does is different from its description. One cannot overemphasize the importance of an accurate function description.